app/code/Magento/Pwa/Test/Mftf/Test/VeniaGuestCheckoutWithMultipleProducts.xml

```diff
@@ -76,7 +87,14 @@
 
         </before>
 
-        <actionGroup ref="ConfirmOrder" stepKey="clickCOnfirmOrderButton"/>
+        <after>
+            <!-- Delete created entity -->
+            <deleteData createDataKey="createCategory1" stepKey="deleteCategory1"/>
+            <deleteData createDataKey="createSimpleProduct" stepKey="deleteSimpleProduct"/>
+            <deleteData createDataKey="createSimpleProduct1" stepKey="deleteSimpleProduct1"/>
+        </after>
+
+        <actionGroup ref="ConfirmOrder" stepKey="clickConfirmOrderButton"/>
 
     </test>
 </tests>
```

Comment:
Instead of commenting on all these, go through and make sure all files have line breaks at the end 😄

Reply:
Updated for all.